### DISCUSSION TITLE
How can I find the way to restore page after I deleted it?

### DISCUSSION THREAD

**Discussion by taosu1024** [requarks/wiki#5046] [Help / Questions]
![image](https://user-images.githubusercontent.com/26999805/155660945-70d56669-6186-44b5-b63e-3c76609e2e21.png)
I didn't find anywhere I can retore page in administration area.
Is it in `recycle bin`? How to enable it?
![image](https://user-images.githubusercontent.com/26999805/155661391-93a4a2ef-0b95-4155-ab2b-a754481c8f40.png)

**Comment by sergiomiracco** [comment]
This is a planned feature.
See here: https://js.wiki/feedback/p/recycle-bin

**Reply by kimegede** [reply]
Made a comment/question on the feedback link about status on this.
